app/code/community/Bolt/Boltpay/Model/FeatureSwitch.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Bolt magento plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2020 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/**
+ * Class Bolt_Boltpay_Model_FeatureSwitch
+ *
+ * The Magento Model class that provides features switches related utility methods
+ *
+ */
+class Bolt_Boltpay_Model_FeatureSwitch extends Bolt_Boltpay_Model_Abstract
+{
+    /**
+     * This method gets feature switches from Bolt and updates the local DB with
+     * the latest values. To be used in upgrade data and webhooks.
+     */
+    public function updateSwitchesFromBolt()
+    {
+        $switchesResponse = $this->boltHelper()->getFeatureSwitches();
+        $switchesData = @$switchesResponse->data->plugin->features;
```

Comment:
is the `@` operator supported on all versons of PHP we support?

Reply:
Yes it is supported on all versions of PHP that we support. 

The `@` operator, in this context has been supported for at least the last 13 years and will likely be supported for the foreseeable future of PHP.